### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by Tekhlay**
### Milestones 💪 

- [x] Run a program using the command line.
- [x] Set up associations between classes and objects.
#### Thank you, reviewers! I value your 🕐.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```
Comment (t-yanick) [inline classroom.rb]: - [ ] Great job working on this milestone. students should only be a setter and not both getter and setter. Kindly refactor as below:
```
attr_accessor :label
attr_reader :students
```

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date)
```
Comment (t-yanick) [inline rental.rb]: - [ ] You should initialize `date`, `person` and `book`. The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. You can read this [example article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more insights

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Tekhlay , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All required classes added
:heavy_check_mark: Great use of getters and setters
:heavy_check_mark: Associations correctly setup

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @Tekhlay  
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Clean and well-organized code
✔️ No Linters error
✔️ Correct Gitflow

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**